sdk/servicebus/service-bus/test/serviceBusClient.spec.ts

```diff
@@ -321,7 +320,7 @@ describe("Test createFromAadTokenCredentials", function(): void {
     "Endpoint=sb://((.*).servicebus.windows.net)"
   ) || "")[1];
 
-  async function testCreateFromAadTokenCredentials(host: string, tokenCreds: any): Promise<void> {
+  async function testCreateFromAadTokenCredentials(host: any, tokenCreds: any): Promise<void> {
```

Comment:
Did the type of the `host` have to change?

Reply:
No, reverted it